README.md

```diff
@@ -130,3 +130,92 @@ example_dat %>%
     ## 7 happy        Yes    138 (57.26%) 137 (59.57%)    0.855 
     ## 8 sugar_factor ""     0.46 (0.3)   0.52 (0.29)     0.0129
     ## 9 age          ""     42 [22, 62]  41 [26, 61]     0.888
+
+#### `format_tbl`
+
+``` r
+univariate <- example_dat %>% 
+  descriptives(
+    variables = c("age", "sugar_factor", "gender", "happiness", "happy")
+  )
+
+univariate
+```
+
+    ## # A tibble: 9 x 3
+    ##   Variable     Label  Statistics  
+    ##   <chr>        <chr>  <chr>       
+    ## 1 gender       female 235 (46.91%)
+    ## 2 gender       male   266 (53.09%)
+    ## 3 happiness    happy  367 (77.75%)
+    ## 4 happiness    sad    105 (22.25%)
+    ## 5 happy        no     105 (22.25%)
+    ## 6 happy        yes    91 (19.28%) 
+    ## 7 happy        Yes    276 (58.47%)
+    ## 8 age          ""     42.3 (22.1) 
+    ## 9 sugar_factor ""     0.49 (0.3)
+
+``` r
+univariate %>%
+  format_tbl()
+```
+
+    ## # A tibble: 12 x 3
+    ##    Variable     Label  Statistics  
+    ##    <chr>        <chr>  <chr>       
+    ##  1 age          ""     42.3 (22.1) 
+    ##  2 gender       ""     ""          
+    ##  3 ""           female 235 (46.91%)
+    ##  4 ""           male   266 (53.09%)
+    ##  5 happiness    ""     ""          
+    ##  6 ""           happy  367 (77.75%)
+    ##  7 ""           sad    105 (22.25%)
+    ##  8 happy        ""     ""          
+    ##  9 ""           no     105 (22.25%)
+    ## 10 ""           yes    91 (19.28%) 
+    ## 11 ""           Yes    276 (58.47%)
+    ## 12 sugar_factor ""     0.49 (0.3)
+
+``` r
+bivariate <- example_dat %>% 
+  descriptives(
+    treatment = "treat",
+    variables = c("age", "sugar_factor", "gender", "happiness", "happy")
+  )
+
+bivariate
+```
+
+    ## # A tibble: 9 x 5
+    ##   Variable     Label  candy         `ice cream`   `P Value`
+    ##   <chr>        <chr>  <chr>         <chr>             <dbl>
+    ## 1 gender       female 124 (48.25%)  110 (45.27%)     0.563 
+    ## 2 gender       male   133 (51.75%)  133 (54.73%)     0.563 
+    ## 3 happiness    happy  185 (76.76%)  181 (78.7%)      0.694 
+    ## 4 happiness    sad    56 (23.24%)   49 (21.3%)       0.694 
+    ## 5 happy        no     56 (23.24%)   49 (21.3%)       0.855 
+    ## 6 happy        yes    47 (19.5%)    44 (19.13%)      0.855 
+    ## 7 happy        Yes    138 (57.26%)  137 (59.57%)     0.855 
+    ## 8 age          ""     42.26 (22.62) 42.39 (21.61)    0.946 
+    ## 9 sugar_factor ""     0.46 (0.3)    0.52 (0.29)      0.0129
+
+``` r
+bivariate %>%
+  format_tbl()
+```
+
+    ## # A tibble: 12 x 5
+    ##    Variable     Label  `candy (N=257)` `ice cream (N=243)` `P Value`
+    ##    <chr>        <chr>  <chr>           <chr>               <chr>    
+    ##  1 age          ""     42.26 (22.62)   42.39 (21.61)       0.946    
```

Comment:
Looks like the cat/cont ordering doesn't hold after running `format_tbl()`.  Can that be preserved?

Reply:
Yeah I noticed this too, and agree that it'd be nice to preserve the ordering from the earlier table. The implementation with `dplyr::group_split()` is elegant, but may need to switch to `base::split()`. I'll work on getting this behavior